index.js

```diff
@@ -1,24 +1,30 @@
-require('dotenv-safe').config(); // Set up environment variables
-const fetch = require('node-fetch'); // Import fetch (for network requests)
-const moment = require('moment');
+import dotenv from 'dotenv-safe';
+import express from 'express';
+import cors from 'cors';
+import generateBillsForSolarProject from './utils/billgeneration';
 
-const EnphaseUtils = require('./utils/enphase.js');
-const UtilityAPI = require('./utils/utility-api.js');
+dotenv.config(); // Set up environment variables
```

Comment:
what environment variables are we using here?

Reply:
I suppose none anymore, but the intention here was that we load environment variables into the app as soon as possible since this is the entry point of the app, as I think is best practice. 

I tried including it in JUST here and getting that to work for the rest of the app, but it wasn’t working. I’ll try again though.. ideally it’s only called in this file and no others